extensions/amp-carousel/0.1/carousel.js

```diff
@@ -155,6 +155,7 @@ export class AmpCarousel extends BaseCarousel {
   doLayout_(pos) {
     this.withinWindow_(pos, cell => {
       this.scheduleLayout(cell);
+      this.scheduleResume(cell);
```

Comment:
Can element's `resumeCallback` be called before layout is complete? Should these be chained? If so, we may need to just add a boolean arg to `scheduleLayout` to follow up layout with resume. Please discuss this with @jridgewell